packages/cli/src/lib/setups/routes/generate.ts

```diff
@@ -119,6 +121,10 @@ export async function generateRoutes(
       ? await getResolvedRoutes()
       : await getResolvedRoutes([options.routeName as RouteKey]);
 
+  if (options.i18nStrategy === 'subfolders') {
+    resolvedRouteFiles.push('($locale)');
+  }
```

Comment:
The problem with this is that we are adding `($locale).tsx` file to the skeleton, when the default skeleton doesn't use locales, right?

Shouldn't we keep the default skeleton without this file, and then create it only when choosing `subfolders`?
Therefore, maybe we should just `writeFile(...)` here with the contents of the `($locale).tsx` file.

Reply:
Honestly I started with that approach, and just wasn't sure which is a better spot to put the file. If you feel like it's better to copy it in manually, I'm happy to change the logic to do so.